include/rpm/rpmkeyring.h

```diff
@@ -107,6 +117,15 @@ rpmPubkey rpmPubkeyLink(rpmPubkey key);
  */
 char * rpmPubkeyBase64(rpmPubkey key);
 
+/** \ingroup rpmkeyring
+ * Return fingerprint of primary key
+ * @param key		Pubkey
+ * @param fp		Fingerprint data
+ * @param fplen		Length of Fingerprint
+ * @return		0 on success
+ */
+int rpmPubkeyFingerPrint(rpmPubkey key, uint8_t **fp, size_t *fplen);
```

Comment:
Fingerprint is a compound word, make that rpmPubkeyFingerprint() instead.

Reply:
Done.